### DISCUSSION TITLE
Revisiting MRE with new updates

### DISCUSSION THREAD

**Discussion by Aamir3d** [MoonRide303/Fooocus-MRE#181] [Ideas]
Hello @MoonRide303 

I really enjoyed Fooocus-MRE.
With the latest version of Fooocus, I find the limited functionality with the lack of metadata and the ability to save JPG hampers my workflow.

Are you planning to revisit these features with MRE and the latest Fooocus updates? I understand that a lot of the earlier MRE features (Controlnet and img-img) are now integrated with Fooocus.

**Comment by MoonRide303** [comment]
@Aamir3d No plans for that at the moment. For my own occasional usage I just stick to last MRE release, as saving metadata and clean UI are more important to me than new features added in v2.1 line.
